server/src/main/java/pl/bk/pizza/store/domain/service/NowProvider.java

```diff
@@ -0,0 +1,13 @@
+package pl.bk.pizza.store.domain.service;
+
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+
+public class NowProvider {
```

Comment:
\[Checkstyle\] ERROR: Utility classes should not have a public or default constructor\.